satellite/satellitedb/accounting.go

```
@@ -72,3 +73,40 @@ func (db *accountingDB) SaveGranulars(ctx context.Context, logger *zap.Logger, l
 	_, err = tx.Update_Timestamps_By_Name(ctx, dbx.Timestamps_Name("LastBandwidthTally"), update)
 	return err
 }
+
+// SaveAtRestRaw records raw tallies of at rest data to the database
+func (db *accountingDB) SaveAtRestRaw(ctx context.Context, logger *zap.Logger, nodeData map[storj.NodeID]int64) error {
+	if len(nodeData) == 0 {
+		logger.Warn("In SaveAtRestRaw with empty nodeData")
+		return nil
+	}
+	tx, err := db.db.Open(ctx)
+	if err != nil {
+		logger.DPanic("Failed to create DB txn in SaveAtRestRaw")
+		return err
+	}
+	defer func() {
+		if err == nil {
+			err = tx.Commit()
+		} else {
+			logger.Warn("DB txn was rolled back in SaveAtRestRaw")
+			err = tx.Rollback()
```

Do not hide the actual error.